app/models/facility.rb

```diff
@@ -79,6 +79,8 @@ class Facility < ApplicationRecord
     facility.validates :name, presence: true
   end
 
+  alias_attribute :block, :zone
```

Comment:
@prabhanshuguptagit I added this to make life easier as we get the language more consistent.  We should be able to migrate to the new field over time via this alias ( and we can do the actual schema rename after the region backfill).